modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -274,6 +275,32 @@ Example output plugin configuration:
     auto_create_group      On
 ```
 
+==== Syslog Configuration Parameters
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Syslog Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | Host name or IP address of the remote Syslog server | Yes |
+| Port | TCP or UDP port of the remote Syslog server | Yes | 514
+| Mode | The mode of the transport type | Yes | udp
+| Syslog format | Syslog protocol format | No | rfc5424
+| Syslog Format | Syslog protocal format | No | 
```

Comment:
Duplicate? Same line as above?

Reply:
Good catch! Thanks

Reply:
Sorry, that was from a merge conflict. I assume the former is the one we want to keep?

Reply:
Resolved.